src/backend/drm/mod.rs

```diff
@@ -0,0 +1,502 @@
+//! Drm/Kms types and backend implementations
+//!
+//! This module provide a `DrmDevice` which acts as a reprensentation for any drm
+//! device and can be used to create the second provided structure a `DrmBackend`.
+//!
+//! The latter represents a crtc of the graphics card you can render to.
+//!
```

Comment:
Also here, a general explanation of how drm works, or a link to the relevant documentation in the drm crate.

(what is a "crtc"? a "connector"? ..)

Reply:
I will most likely go with a link and do a PR with more explanations on the drm crate.

Reply:
That sounds good 👍